content/blog/almalinux-accounts-launch.md

```diff
@@ -0,0 +1,91 @@
+
+## Why?
+Introducing our Single Sign-On (SSO) system offers numerous benefits for both users and the AlmaLinux OS Foundation. SSO enhances security, streamlines access, and simplifies user management across our ecosystem. With a single set of credentials, users can seamlessly navigate all AlmaLinux websites and services. This unified login experience ultimately boosts productivity, fosters collaboration, and provides a more cohesive and personalized experience for our community.
+
+## Software Stack
+In the spirit of FOSS, it was only natural for us to stick with a free and open-source software stack. The chosen components should be familiar to anyone who has dealt with directory management or SSO in the Enterprise Linux space.  We took inspiration from Fedora's account system for some components.
+
+We have chosen the following FOSS components to power our SSO:
+
+- [FreeIPA](https://www.freeipa.org) for directory and user management.
+- [Noggin](https://github.com/fedora-infra/noggin), a web front-end for user self-service. We've been doing work in the Fedora project to get noggin and it's dependencies added to EPEL 9. This will benefit anyone looking to deploy noggin on RHEL 9 or RHEL 9 compatible distros. <sup>1</sup>
+- [Keycloak](https://www.keycloak.org/), an OpenID Connect system to sit between applications and LDAP.  We've started contributing to a playbook mostly maintained by RHEL engineers to help ease the deployment of Keycloak on EL distros. <sup>2</sup>
+
+## What About My Existing Account(s)?
+We have a transition plan for the next few weeks to get all systems fully supporting our [AlmaLinux Accounts](https://accounts.almalinux.org) and all existing accounts transitioned into the system, or otherwise integrated.
+
+### chat.almalinux.org (Mattermost)
+chat.almalinux.org is the single largest standalone user database we have to transition.  Luckily this is a very easy and painless task for us as well as users.
+
+After you've created your [AlmaLinux Account](https://accounts.almalinux.org), navigate to the `Profile -> Security` section in Mattermost.
+
+![Step 1 of converting MatterMost account to use AlmaLinux Accounts](/blog-images/almalinux-accounts-mattermost-step1.png)
+
+From here click on `Edit` next to `Sign-in Method` and then `Switch to Using AD/LDAP` and you can link your new AlmaLinux account.  *Keep in mind that once you link your account you will always use your AlmaLinux account to sign in to chat.almalinux.org.*
+
+![Step 2 of converting MatterMost account to use AlmaLinux Accounts](/blog-images/almalinux-accounts-mattermost-step2.png)
+
+![Step 3 of converting MatterMost account to use AlmaLinux Accounts](/blog-images/almalinux-accounts-mattermost-step3.png)
```

Comment:
Need to include a screenshot of the screen where you enter your passwords, with the right info flagged, so folks know where to put what.